pg_auto_failover.spec

```diff
@@ -0,0 +1,66 @@
+%global pgmajorversion 11
+%global pgpackageversion 11
+%global pginstdir /usr/pgsql-%{pgpackageversion}
+%global sname pg_auto_failover
```

Comment:
In the blog post I had used `pg-auto-failover-10_11` as the package name to be installed. To keep it simple, could we use "pg-auto-failover" for package names?

Reply:
I used the `_` here to stay consistent with [pg_cron](https://github.com/citusdata/packaging/blob/redhat-cron/pg_cron.spec#L4) but I am ok with this as well.

your blogpost needs to change a bit I think as the package I get is named like this: `pg-auto-failover_10-1.1.0.citus-0.0.pre.20190502.76c4628.el7.x86_64.rpm`
Don't pay attention to `1.1.0` that will become `1.0.0` when we release. but the pg version is separated with a `_`, guess due to line 9-10.

Reply:
guess your name is actually correct, citus-ha packages were having the version in the releases as well, I based this release script on the ones we used there.